### ISSUE TITLE
Licensing

### ISSUE THREAD

**Issue by brian-ubi** [typora/typora-issues#4875]
I am a Linux user. I had asked support some time ago and you were not clear on if Linux would be included in the paid only model, but it seems that it has. I wanted to comment on that, respectfully.

While I understand that some companies produce proprietary software and often decide to charge for it. I understand this. As a Linux user, I always hesitate to use software that is not FOSS, but sometimes there is really nice product that causes an expectation. Typora has been one of those softwares. However, seeing that you are requiring activation, you lost me right there, all else aside. I will not be a part of "trackers" installed on my system, and many, maybe most, Linux users feel the same way... one of the main reasons that some people move away from Mac and Windows, to remove the company user spying actions. On Linux, while one software might be nice, we always have a good number of FOSS solutions that do not place such things on our computers.

Things are absolutely different with softwares that directly interact with online systems, there needs to be a connection established in order to make the thing work. This is a text editor. It may well be that you are not targeting Linux users, in which case this makes sense. I just wanted to share the opinion of a now former Typora user on Linux.

**Comment by Iipal** [comment]
Totally agree with you. But I don't really care about connection or licensing the software but that the software that I love from first sight now requires to be paid. I really was in love with the Typora, I even use it in daily use as my _fancy notebook_ and recommended to everyone at the work but now I can't. 15$ Is not that big of a deal, but I don't want to pay for the software that was free up to this moment and what is not that different from other free markdown editors.

At some moment I rather make a donation of 50-100$, just what I did with Sublime Text, they actually do not restrict you to using the software as you wish, but just remind you to be a bit thankful for the devs and I was

**Comment by tobx** [comment]
I agree with @brian-ubi, but I want to mention another point. I opened the issue #3681 that was a duplicate of #3541 that was a duplicate of #1638. The ladder is open for almost 3,5 years now. I guess most people using markdown work IT related so I really do not understand how this is not a major issue. Not being able to edit markdown in source mode with proper syntax highlighting is a showstopper for me, so I am using other products, but maybe I am the exception...

I am not in a position to judge companies' pricing model decisions, but I am currently in the position that I do not use the software because of bugs and now I cannot even really easily check if the bugs are fixed without paying :/

@lipal I think Typora was free long enough for the developers to know how much money they get with donations. It was obviously not enough to make it work for them and you can not expect people to develop for free. If they do it, fine, but not everyone can do this forever. Sublime Text also requires a license, but they do not actively stop you from using it without. Very cool, but I guess this is not working for every piece of software.

**Comment by abnerlee** [comment]
Please check https://support.typora.io/purchase/#can-i-use-typora-for-free-

For linux version you will have a unlimited trial time. It should display "not now" instead of "start 15 days trial" on Linux, but we fixed in 1.0.3

For other versions, payment will be required. But it is your choice, after all.

**Comment by Iipal** [comment]
@abnerlee today I just updated the app, and now I have only 1 day left ?
![image](https://user-images.githubusercontent.com/31858171/144643496-3004b4ba-c21c-443b-96e2-1766d15290a1.png)


NICE

**Comment by brian-ubi** [comment]
I have no issues with being tankful to developers. I am for sure. The way it works on Linux is you use free software and you provide or assist in the creation of free software, using that software. I use FOSS for almost everything. I do work on one paid applications, but I contribute where I can to development of FOSS, which does take my time. For me it is mostly bug reporting and testing. I pay with my time. I have communicated with many far better developers than me and it is most always a nice conversation. All are polite and wanting to hear thoughts to improve, and I enjoy the experience.

In response to comments, I am well aware of Sublime and how they handle this, both for Text and Merge. It is not so bad. Similar to others like GitKraken and SmartGit... you are basically shown a message that you are using the software for free. It sort of seems that my post was taken negatively.. that was not my intent. I understand that different businesses structure themselves in different ways in order to gain success. My opinion... I think Typora is a reasonable software. What I used it for is to quickly view a MD file. If I am editing that file, it works good only for writing. Coding, not really. However, if I just want to do basic work, it is perfect. If I want to be more involved, I will use GEdit (yes, a text editor with plugin), Atom, VSCode, GhostWriter, etc. That said, what I like most about Typora is the layout and interface, as it applies to writing. It is clean and easy. I do not need functionality for that, it serves its purpose.

As @abnerlee stated, it is a choice, and I respect that. If you want to push payment on Linux users vs just allowing it on a free version, that is your choice. I have nothing against that. I am happy to use your software and continue to recommend it as long as it is FOSS friendly. At this time, it is just not. Typora is one of VERY few proprietary softwares I had installed on my systems. That says a lot for your product. However, now it is clear that you have included code to communicate back to you that I cannot turn off. This is a violation of privacy, but again, you are producing proprietary software, so it is somewhat expected.

So... I will try it again once it is "fixed". I am happy to. Until then, as a Linux and FOSS user, I do not and will not support "trackers" placed on my system, especially from a proprietary software where I cannot see what is actually happening. While some may be ok with this, those in IT security will see many red flags here. To all their own.

**Comment by brian-ubi** [comment]
I wanted to add... I do not know nor am I going to lookup with of these work on Windows or Mac... but we have many alternatives... Atom, Emacs, Remarkable, Haroopad, ReText, UberWriter, Mark My Words, Vim (with plugin), GEdit (with plugin), Brackets (with plugin), GitBook, Abricontine, Zrttlr, Joplin, etc... This list could go on for a long time.

It is my opinion that there is no need for paid Markdown Editors when there is a large group of existing and FOSS that supports this functionality. As I stated, I used Typora only for quick viewing and sometimes basic editing. There are plenty of viewing options (I wont even go into that list), and the editing, well, I used Typora only because it is lightweight and opened quickly for a fast edit. Can I respectfully ask... does that sound like a software I would pay for? In contract, where do I really spend time and see value I would pay for? Atom, VSCode, Zettlr and Joplin. However, when it comes to just opening a MD file for a quick edit, Atom and VSCode are heavy to load and Zettlr and Joplin are not really designed to open and edit single files.

I think Typora was made with the user in mind and it very friendly. I hope this model works for you. I mean that with full respect. For me, in my list above, I left out Sublime Text (with plugin) as I don't use it, it is "nuisanceware". I fail to understand why softwares that are not best in class do this while to actual best in class are FOSS, but to each their own. I am sure that users of Windows and Mac see this differently. Both Typora and Sublime are **great** softwares and I would happily use both. I just hope that the developers of these that use FOSS to do the development are contributing money in return to the developers that make the software they use in return.
